server_addon/fusion/server/settings.py

```diff
@@ -105,6 +132,22 @@ class FusionSettings(BaseSettingsModel):
                 "farm_rendering"
             ],
             "image_format": "exr"
+            "image_format": "exr",
+            "default_frame_range_option": "asset_db"
+        },
+        "CreateImageSaver": {
+            "temp_rendering_path_template": "{workdir}/renders/fusion/{product[name]}/{product[name]}.{ext}",
+            "default_variants": [
+                "Main",
+                "Mask"
+            ],
+            "instance_attributes": [
+                "reviewable",
+                "farm_rendering"
+            ],
+            "image_format": "exr"
```

Comment:
```suggestion
```

Reply:
this is causing server add-on to crash during booting up server

Reply:
Wrong merge, fixed.